packages/eslint-config-custom/react.js

```diff
@@ -97,6 +97,18 @@ module.exports = {
         },
       },
     ],
+    "@typescript-eslint/no-unused-vars": [
+      "error",
+      {
+        args: "all",
+        argsIgnorePattern: "^_",
+        caughtErrors: "all",
+        caughtErrorsIgnorePattern: "^_",
+        destructuredArrayIgnorePattern: "^_",
+        varsIgnorePattern: "^_",
+        ignoreRestSiblings: true,
+      },
+    ],
```

Comment:
Not sure why we never had this before but you can ignore this rule by prefixing vars we never intend to use with an underscore now `_`

Reply:
Is this not just replacing one ignore statement with another? Ignore unused var -> ignore underscore dangle
😕 

![image](https://github.com/user-attachments/assets/c9b1cd8a-7313-4b0b-8eec-b7196065c312)

Reply:
Only here since it is a proper var. Honestly this whole type gymnastics we are doing here is a bit odd and we should revisit it but for now, adding an ignore is probably sufficient.

Reply:
I think it is my type gymnastics 🥹